src/interface/efi/efi_veto.c

```diff
@@ -30,6 +30,7 @@ FILE_LICENCE ( GPL2_OR_LATER_OR_UBDL );
 #include <ipxe/efi/Protocol/LoadedImage.h>
 #include <ipxe/efi/Protocol/ComponentName.h>
 #include <ipxe/efi/efi_veto.h>
+#include <stdio.h>
```

Comment:
That `include` should probably be avoided.

Reply:
my bad, it's removed now